.travis.yml

```diff
@@ -12,7 +12,7 @@ cache:
 language: python
 python:
   - "2.7"
-  - "3.3"
+  - "3.4"
```

Comment:
You also need to change Theano/.travis/travis_install.sh to create a conda environment for python 3.4.